Makefile

```diff
@@ -1,6 +1,6 @@
 NAME = heckler
 BUILD_GIT_SHA = $(shell git rev-parse --short HEAD)
-IMAGE = dockerhub.com/lollipopman/$(NAME)
+IMAGE = dockerhub.com/heckler/$(NAME)
```

Comment:
Does this actually work? I don't believe there is currently a heckler user for dockerhub, and I can't seem to find the image referenced.

Reply:
I was just wondering the same thing.

Reply:
We are not pushing any images to the public docker hub. For local testing and builds we can have any "org" name.
I validated all make commands , able to build binaries with this change

Reply:
perhaps add a comment indicating that the org name doesn't matter for localdev?

Reply:
yes